src/utils/envVars.ts

```diff
@@ -47,7 +47,10 @@ if(process.env.REACT_APP_PRICE_PER_VALIDATOR && Number.isNaN(Number(process.env.
 }
 export const PRICE_PER_VALIDATOR        = process.env.REACT_APP_PRICE_PER_VALIDATOR || 32;
 
-export const EJECTION_PRICE = 16;
+if(process.env.REACT_APP_EJECTION_PRICE && Number.isNaN(Number(process.env.REACT_APP_EJECTION_PRICE))) {
```

Comment:
Should we add this same check for `PRICE_PER_VALIDATOR`?

Reply:
Should be right above it already (L45-47), or am I missing something

Reply:
Aha, whoops my bad. The Github PR didn't show those lines 😅